.env-ui.sample

```diff
@@ -1,2 +1,3 @@
+BLOCK_EXPLORER_URL=https://mumbai.polygonscan.com
```

Comment:
This and the local UI dev equivalent needs to be documented in the root README.

Reply:
This also needs to be `ISSUER_UI_BLOCK_EXPLORER_URL` to remain consistent with the rest of the env vars.